client/src/components/History/Multiple/MultipleViewItem.vue

```diff
@@ -42,7 +60,38 @@ function onViewCollection(collection: object) {
         </div>
     </div>
 
-    <div v-else id="list-item" class="d-flex flex-column align-items-center w-100">
+    <div v-else id="list-item" class="d-flex flex-column w-100">
+        <div class="d-flex justify-content-between align-items-center">
+            <div>
+                <BButton
+                    size="sm"
+                    class="my-1"
+                    :disabled="sameToCurrent || history.purged || history.archived"
```

Comment:
```suggestion
                    :disabled="sameToCurrent"
```

In theory, you can now switch to any of these histories :)